sos/plugins/docker.py

```diff
@@ -77,10 +77,18 @@ def setup(self):
         if self.get_option('all'):
             ps_cmd = "{0} -a".format(ps_cmd)
 
-        result = self.get_command_output(ps_cmd)
-        if result['status'] == 0:
-            containers = [c for c in result['output'].splitlines()]
-            for container in containers:
+        img_cmd = '{0} images -q'.format(self.docker_cmd)
+        insp = []
+
+        for icmd in [ps_cmd, img_cmd]:
+            result = self.get_command_output(icmd)
+            if result['status'] == 0:
+                insp += [c for c in result['output'].splitlines()]
+
+        insp = list(set(insp))
+
```

Comment:
Why not make `insp` a set in the first place? Then you can just `list(insp)`. You'd need to `.add()` rather than `+=` but I think it aids readbility.